lesson25/src/js/validation.js

```diff
@@ -0,0 +1,45 @@
+export const showErrorMessage = (target,message) => {
+    target.classList.add("border-2","border-red-500");
+
+    const errorMessage = document.createElement("p");
+    errorMessage.classList.add("field-invalid","text-sm","my-2","text-rose-600");
+	errorMessage.textContent = message;
+
+	if(!target.nextElementSibling){
+		target.insertAdjacentElement("afterend",errorMessage);
+	}
+}
+
+export const removeErrorMessage = (target) => {
+    target.classList.remove("border-red-500");
+
+    const error = target.nextElementSibling;
+    if(error && error.classList.contains("field-invalid")){
+        error.remove();
+    }
+}
+
+export const checkRequired = (input) => {
```

Comment:
nits 
I found this function name a bit confusing.
I think it is the same for all inputs that it is a required check.
It would be nice to know that it is processing when the input field is empty.

Reply:
Thank you for your comment.
Yes, I thought the name was confusing.

I've changed the method of checking required and blank fields directly in main.js instead of making it a function.
https://github.com/nakagawamai/js-lesson/pull/27#discussion_r1051318390